index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
```

Comment:
- [OPTIONAL] Kindly add the hover effect to the `Get in touch` button in the contact section as requested in the Figma design guidelines.